shared/schema.ts

```
@@ -75,6 +75,54 @@ export type CronJob = {
+  memoryUsage: integer("memory_usage"), // Percentage
+  temperature: integer("temperature"),
+  diskReadSpeed: integer("disk_read_speed"), // KB/s
+  diskWriteSpeed: integer("disk_write_speed"), // KB/s
+  networkRx: integer("network_rx"), // KB/s
+  networkTx: integer("network_tx"), // KB/s
+});
+
+export type HistoricalMetric = typeof historicalMetrics.$inferSelect;
+export type InsertHistoricalMetric = typeof historicalMetrics.$inferInsert;
+
+export type SystemInfo = SystemInfo & {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'SystemInfo'. Consider to delete it or rename it.

'SystemInfo' is defined here:


(lint/suspicious/noRedeclare)